pom.xml

```diff
@@ -36,6 +36,15 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.vladmihalcea</groupId>
+			<artifactId>hibernate-types-52</artifactId>
```

Comment:
what is it? why you need that?

Reply:
deleted, I need this for @Type for using json, at now, i found another solution to solve this problem